scargo/commands/check.py

```diff
@@ -186,15 +186,25 @@ def check(self) -> None:
         super().check()
 
     def check_file(self, file_path: Path) -> CheckResult:
+        copyrights = self.copyright_desc.split("\n")[:-1]
+        slice_present = 0
+        copyright_present = False
         with open(file_path, encoding="utf-8") as file:
             for line in file.readlines():
-                if self.copyright_desc in line:
-                    return CheckResult(problems_found=0)
-                if "copyright" in line.lower():
-                    logger.warning(
-                        "Incorrect and not excluded copyright in %s", file_path
-                    )
-                    return CheckResult(problems_found=1, fix=False)
+                for copyrights_slice in copyrights:
+                    if copyrights_slice in line:
+                        slice_present += 1
+                    if slice_present == len(copyrights):
```

Comment:
You're only checking that all lines are present somewhere in the file, not that they're consecutive. It can be improved.